python/friesian/example/recall/search.py

```diff
@@ -135,14 +130,26 @@ def search(args):
 
     parser.add_argument('--num_threads', type=int, default=8,
                         help='Set the environment variable OMP_NUM_THREADS for each faiss task')
-    parser.add_argument('--cluster_mode', type=str, default='yarn',
-                        help='The cluster mode, such as local, yarn')
+    parser.add_argument('--num_repartition', type=int, default=12,
+                        help='The number of repartition')
+    parser.add_argument('--cluster_mode', type=str, default='local',
+                        help='The cluster mode, such as local, yarn or spark-submit')
+    parser.add_argument('--num_nodes', type=int, default=4,
+                        help='The number of nodes to use in the cluster.')
+    parser.add_argument('--executor_cores', type=int, default=8,
+                        help='The executor core number')
+    parser.add_argument('--executor_memory', type=str, default="50g",
+                        help='The executor memory')
+    parser.add_argument('--driver_cores', type=int, default=4,
+                        help='The driver core number')
+    parser.add_argument('--driver_memory', type=str, default="36g",
+                        help='The driver memory')
```

Comment:
What about removing these two arguments and just give default values?
driver memory doesn't need that much right?

Reply:
For consistency with the other examples, I think we should keep these parameters.

Reply:
removed arguments of driver